### PR TITLE
Radar blip colors now respect the IFF color config option. Also added…

### PR DESCRIPTION
**Pull request by testtank400**
… another palette enum to differentiate palette access between color functions

### CODE REVIEW

File: xwa_hook_tourmultiplayer/zip/hook_tourmultiplayer.cfg [hunk 1]
```diff
@@ -16,8 +16,3 @@ ChangeCombatSimMusic = 1
 ; TourOfDutyButtonRequiredMissionIndex = value
 ; Define the index of the mission when the Tour of Duty button become active.
 TourOfDutyButtonRequiredMissionIndex = 1
-
-; GreenAndRedForIFFColorsOnly = 0 (default) or 1
-; When set to 0, use IFF colors
-; When set to 1, change IFF colors to green for friendlies and red for enemies
-GreenAndRedForIFFColorsOnly = 0
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/zip/hook_tourmultiplayer.cfg]: Why remove this setting?

**Review by JeremyAnsel — COMMENTED** [review]
One comment on the config file

File: xwa_hook_tourmultiplayer/zip/hook_tourmultiplayer.cfg (hunk 1 above)
Comment (testtank400) [reply xwa_hook_tourmultiplayer/zip/hook_tourmultiplayer.cfg]: I'm not sure how that happened. Something must have gotten lost during the sync.

Comment (testtank400) [reply xwa_hook_tourmultiplayer/zip/hook_tourmultiplayer.cfg]: Ok it's fixed now.

**Pull request merged**